### PR TITLE
ENG-416

### PR DESCRIPTION
**Pull request by firegloves**
- content-type usage details logic update
- tests update

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/ContentTypeService.java [hunk 1]
```diff
@@ -179,9 +179,34 @@ public Integer getComponentUsage(String componentCode) {
...
+            return "Published";
+        } else if (contentDto.getStatus().equals(Content.STATUS_READY)) {
+            if (contentDto.isOnLine()) {
+                return Content.STATUS_PUBLIC + " ≠ " + Content.STATUS_READY;
+            } else {
+                return Content.STATUS_READY;
+            }
+        } else {
+            if (contentDto.isOnLine()) {
+                return Content.STATUS_PUBLIC + " ≠ " + Content.STATUS_DRAFT;
```
Comment (joewhite101) [inline plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/ContentTypeService.java:206]: What are we using this string for? Are we going to run into encoding problems with "≠" ? Seems like we might be better off with a standard string, especially if this is coming back in an API result.

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by Kerruba** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by Kerruba** [comment]
Retest this please

**Comment by entando-service** [comment]
All is Well

**Pull request merged**